lib/rubill/base.rb

```diff
@@ -48,9 +48,14 @@ def self.delete(id)
       Query.delete(remote_class_name, id)
     end
 
-    def self.where(filters=[])
+    def self.where(filters=[], hash_filters={})
+      filters, hash_filters = [], filters if filters.is_a?(Hash)
```

Comment:
I don't like this pattern. What do you think about changing the API here to accept a hash instead of an array of Filter objects?